src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -688,7 +690,7 @@ void FullyConnected::initSupportedPrimitiveDescriptors() {
     for (auto& desc : descs) {
         auto first_desc = dnnl::primitive_desc(DnnlExtensionUtils::clone_primitive_desc(desc.get()));
 
-        const bool first_match = customImplPriorities.empty();
+        const bool first_match = !customImplPriorities.empty();
```

Comment:
The same: could you please share details thy this change is needed?

Reply:
customImplPriorities vector is the implement parsed via Ngraph node RTinfo.  

When there is no customImplPriorities:
Current first_match behavior in master branch would only add one dnnl::primitive_desc for one memory layout type and this  dnnl::primitive_desc implement type is based on ONEDNN list priority. For example, for fp32 current master code would only add one implement type(brgemm av512) for nhwc layout. Such as, For INT8 model, it is found the implement type list priority can't work when we want kernel type jit_gemm priority higher than avx2 brgemm. Because only  brgemm_avx2 is added into the dnnl::primitive_desc for AVX2_VNNI ISA platforms. 

I recalled the former code would add all the supported dnnl::primitive_desc implement type(brgemm av512, brgemm avx2 and  other) into the supportedPrimitiveDescriptor list. So this changes would added all supported into.  Only by this way, selectPreferPrimitiveDescriptor() can work as OV priority list.
When there is customImplPriorities parsed from RT info:
No need to add all supported dnnl::primitive_des. Only added the implement type (can be found in the  customImplPriorities ) into list.

So that is why made this changes.